chain/client/src/stateless_validation/partial_witness/partial_witness_actor.rs

```diff
@@ -268,11 +268,8 @@ impl PartialWitnessActor {
         let encoder = self.witness_encoders.entry(chunk_validators.len());
         let (parts, encoded_length) = encoder.encode(&witness_bytes);
 
-        Ok(chunk_validators
-            .iter()
-            .zip_eq(parts)
-            .enumerate()
-            .map(|(part_ord, (chunk_validator, part))| {
+        chunk_validators.iter().zip_eq(parts).enumerate().for_each(
```

Comment:
super nit: since we don't need to call a `collect`, we can probably do a simple for loop instead of `for_each`